.eslintrc

```diff
@@ -37,3 +37,4 @@ rules:
   no-unused-expressions: [0]
   no-use-before-define: [2, nofunc]
   yoda: [0]
+  linebreak-style: [0]
```

Comment:
I don't know much about this, but I didn't change anything or even open a single file in the editor when I tried running `npm start` the first few times (so there's no chance of the editor accidentally changing LF to CRLF). But eslint kept throwing those errors.